src/main/kotlin/org/wfanet/measurement/kingdom/db/ReportDatabase.kt

```diff
@@ -0,0 +1,83 @@
+package org.wfanet.measurement.kingdom.db
+
+import kotlinx.coroutines.flow.Flow
+import org.wfanet.measurement.common.identity.ExternalId
+import org.wfanet.measurement.internal.kingdom.Report
+import org.wfanet.measurement.internal.kingdom.Report.ReportState
+import org.wfanet.measurement.internal.kingdom.ReportConfigSchedule
+import org.wfanet.measurement.internal.kingdom.ReportDetails
+import org.wfanet.measurement.internal.kingdom.ReportLogEntry
+import org.wfanet.measurement.internal.kingdom.Requisition
+import org.wfanet.measurement.internal.kingdom.Requisition.RequisitionState
+
+/**
+ * Abstraction around the Kingdom's relational database for storing [Report] metadata.
+ *
+ * TODO: with v2alpha API, rename to "MeasurementDatabase".
```

Comment:
Are we going to replace these Report, ReportDetails etc protos to Measurement, MeasurementDetails?

Reply:
Eventually, I think it would make sense to do this.